packages/commonjs/test/fixtures/form/ignore-ids-function/output.js

```diff
@@ -1,7 +1,8 @@
+import * as commonjsHelpers from '_commonjsHelpers.js';
 import 'bar';
 import bar from '_bar?commonjs-proxy';
 
-var foo = require( 'foo' );
+var foo = commonjsHelpers.commonjsRequire( 'foo' ,"./fixtures/form/ignore-ids-function");
```

Comment:
We need a detailed explanation in the PR body as to why these fixtures were changed to such a degree. Typically when adding new features, the existing fixtures should be left as-is and tests should pass. If they don't, then that's a sign that the feature is breaking.  You've checked "no" for Breaking Changes, so these fixtures really shouldn't be modified.

Reply:
I remember there was a very valid reason, but I'll look into it again to make sure I know what I'm talking about when I'm talking about it :-)

Reply:
Okay, reverted it for 3 tests, and another 3 tests are problematic for a valid reason. Adding notes to the PR.